ofs/apps/cpeng/hps.cpp

```diff
@@ -92,6 +101,10 @@ class cpeng : public opae::afu_test::command
       ->transform(CLI::AsNumberWithUnit(units));
     app->add_option("-c,--chunk", chunk_, "Chunk size. 0 indicates no chunks")
       ->default_str(std::to_string(chunk_));
+    app->add_option("-r,--data-request-limit",
```

Comment:
Do you need to bind data_request_limit_ to the command option here?

Reply:
Yes, will fix